pkgs/by-name/si/signal-export/package.nix

```diff
@@ -25,6 +26,8 @@ python3.pkgs.buildPythonApplication rec {
     emoji
     markdown
     pysqlcipher3
+    rich
```

Comment:
Why do we need rich here? I don't see it in https://github.com/carderne/signal-export/blob/main/pyproject.toml

Reply:
:+1: 
We get it indirectly via typer. I found it in requirements.lock and didn't pay attention that it is an indirect dependency. I remove it.